src/main/webapp/css/main.css

```diff
@@ -168,3 +168,69 @@ button.remove-user-button {
   color: white;
   font-weight: lighter;
 }
+
+.jasmineBio {
```

Comment:
As I mentioned above, you only need one copy of the bio class here, and you can reuse it for all three bios. You also don't need the `.success` classes down here.